src/reactapp/src/pages/EvaluationForm.js

```diff
@@ -0,0 +1,185 @@
+import {
+    Box,
+    chakra,
+    Container,
+    Stack,
+    Text,
+    Image,
+    Flex,
+    VStack,
+    Button,
+    Heading,
+    SimpleGrid,
+    StackDivider,
+    useColorModeValue,
+    VisuallyHidden,
+    List,
+    ListItem,
+    HStack,
+    FormControl,
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    Select,
+    Input,
+    ButtonGroup,
+} from "@chakra-ui/react";
+import "../App.css";
+import React, { useEffect, useState } from "react";
+
+export default function EvaluationForm() {
+  const [evaluationObjectives, setEvaluationObjectives] = useState([]);
+
+  useEffect(() => {
+    fetchEvaluationObjectives();
+  }, []);
+
+  // Retrieve evaluation objectives from API
+  // Needed to fetch to localhost:7065 with the full URL to avoid CORS issues?
+  const fetchEvaluationObjectives = async () => {
+    try {
+      const response = await fetch("https://localhost:7065/api/EvaluationApi");
```

Comment:
Please add a function to the FetchHelpers.js for a GET request, if there isn't one yet. Then you can see the other examples of how to add the authorization header to the request. Also, you'll see a note about eventually moving away from the hard-coded localhost base URL.